packages/scene-composer/src/components/WebGLCanvasManager.tsx

```diff
@@ -104,6 +89,9 @@ export const WebGLCanvasManager: React.FC = () => {
   }, [gridHelperRef.current]);
 
   useEffect(() => {
+    setCursorVisible(!!addingWidget);
+    setCursorStyle(addingWidget ? 'edit' : 'move');
```

Comment:
This should probably be handled in state, and not in the WebGLCanvasManager.

Reply:
Good call out.